### PR TITLE
Update ont-nokia-g-010g-a.md

### PR DESCRIPTION
**Pull request by drizuid**
added chipset, taken directly from my g-010g-a

### CODE REVIEW

**Comment by nanomad** [comment]
Hi @drizuid do you have an internal photo that shows this chip being in use? Alternatively a boot or similar is sufficient as well

**Comment by simonebortolin** [comment]
Hi @drizuid and thank you for your first contribution and we are very happy to accept it, but you can have a internal photo or alternatively photos do you by any chance access to ssh/telnet/serial on the nokia g-010g-a?

**Comment by drizuid** [comment]
**Box itself**
box
**Info screen**
att pon

You guys will see there are some other fields you have blank that I have data for, but since those can vary by region and customer, I didn't feel it was appropriate for me to populate them.

**Comment by drizuid** [comment]
I should also note that this MAY BE the southeast former bellsouth region box (usually indicated by the "alt optic" sticker) 
some notes on this
https://www.dslreports.com/forum/r33456904-
https://www.dslreports.com/forum/r33458733-

File: _ont/ont-nokia-g-010g-a.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ parent: Nokia
 | -------------- | -------- |
 | Vendor/Brand   | Nokia    |
 | Model          | G-010G-A |
-| Chipset        |          |
+| Chipset        | Broadcom BCM6838  |
 | Flash          |          |
```
Comment (simonebortolin) [inline _ont/ont-nokia-g-010g-a.md:15]: @drizuid Yes, but you should also fix the whole table by adding spaces. Mine was just a suggestion.

**Comment by simonebortolin** [comment]
@drizuid 

> I should also note that this MAY BE the southeast former bellsouth region box (usually indicated by the "alt optic" sticker) I do not know if this leads to any physical hardware differences aside from the actual fiber
> some notes on this
> https://www.dslreports.com/forum/r33456904-
> https://www.dslreports.com/forum/r33458733-

I highly doubt the hardware part is different, most likely the BOSA changes.

a note should certainly at the end of page we need to add the section Miscellaneous Links with DSLreport link.

**Review by simonebortolin — APPROVED** [review]

**Pull request merged**